timelock-impl/src/main/java/com/palantir/atlasdb/timelock/TimeLockResource.java

```diff
@@ -89,17 +89,21 @@ TimeLockServices getOrCreateServices(String namespace) {
         return servicesByNamespace.computeIfAbsent(namespace, this::createNewClient);
     }
 
-    public int numberOfClients() {
+    public int getNumberOfActiveClients() {
         return servicesByNamespace.size();
     }
 
+    public int getMaxNumberOfClients() {
+        return maxNumberOfClients.get();
+    }
+
     private TimeLockServices createNewClient(String namespace) {
         Preconditions.checkArgument(!namespace.equals(PaxosTimeLockConstants.LEADER_ELECTION_NAMESPACE),
                 "The client name '%s' is reserved for the leader election service, and may not be "
                         + "used.",
                 PaxosTimeLockConstants.LEADER_ELECTION_NAMESPACE);
 
-        if (numberOfClients() >= maxNumberOfClients.get()) {
+        if (getNumberOfActiveClients() >= maxNumberOfClients.get()) {
             log.error(
                     "Unable to create timelock services for client {}, as it would exceed the maximum number of "
                             + "allowed clients ({}). If this is intentional, the maximum number of clients can be "
```

Comment:
Not a part of the original change - but could we change the `maxNumberOfClients` in the `SafeArg` below to be `getMaxNumberOfClients()`? Right now it looks like we log the supplier, which isn't helpful